Cargo.toml

```diff
@@ -12,6 +12,20 @@ keywords = ["bitcoin", "paper", "wallet", "offline"]
 readme = "README.md"
 
 [dependencies]
-bitcoin = { version="0.25.2", features = ["rand"] }
+bitcoin = { version="0.26", features = ["rand"] }
+miniscript = "5.1.0"
 qr_code = { version="1.0.0", features = ["bmp"] }
-base64 = "0.13.0"
\ No newline at end of file
+base64 = "0.13.0"
+structopt = "0.3"
+rand = "0.8"
+log = "^0.4"
+env_logger = "0.8"
+maud = "*"
```

Comment:
i would pin the latest version here, just in case in the future the api changes

Reply:
done in 2bad953